src/test/java/com/puppycrawl/tools/checkstyle/checks/blocks/LeftCurlyCheckTest.java

```diff
@@ -250,15 +250,17 @@ public void testLineBreakAfter() throws Exception {
             "20:17: " + getCheckMessage(MSG_KEY_LINE_PREVIOUS, "{", 17),
             "26:22: " + getCheckMessage(MSG_KEY_LINE_BREAK_AFTER, "{", 22),
             "28:17: " + getCheckMessage(MSG_KEY_LINE_PREVIOUS, "{", 17),
-            "35:33: " + getCheckMessage(MSG_KEY_LINE_BREAK_AFTER, "{", 33),
-            "36:21: " + getCheckMessage(MSG_KEY_LINE_BREAK_AFTER, "{", 21),
-            "39:29: " + getCheckMessage(MSG_KEY_LINE_BREAK_AFTER, "{", 29),
-            "39:34: " + getCheckMessage(MSG_KEY_LINE_BREAK_AFTER, "{", 34),
-            "45:37: " + getCheckMessage(MSG_KEY_LINE_BREAK_AFTER, "{", 37),
-            "51:12: " + getCheckMessage(MSG_KEY_LINE_BREAK_AFTER, "{", 12),
-            "54:5: " + getCheckMessage(MSG_KEY_LINE_PREVIOUS, "{", 5),
-            "56:19: " + getCheckMessage(MSG_KEY_LINE_BREAK_AFTER, "{", 19),
-            "66:1: " + getCheckMessage(MSG_KEY_LINE_PREVIOUS, "{", 1),
+            "30:21: " + getCheckMessage(MSG_KEY_LINE_PREVIOUS, "{", 21),
```

Comment:
Such changes in test are hard to validate, please make a separate test with a separate input for new test cases.

Reply:
done